supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesInitializationActor.scala

```diff
@@ -0,0 +1,157 @@
+package cromwell.backend.impl.jes
```

Comment:
Created https://github.com/broadinstitute/cromwell/issues/722 to perform this change in another PR due to several changed/new files in this PR.